ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -247,6 +248,17 @@ public async Task<IEnumerable<NotificationBannerModel>> GetNotificationBannerMod
                 .Select(n => new NotificationBannerModel(n, showLink: true));
         }
 
+        public async Task<IEnumerable<Notification>> GetInactiveNotificationsToCloseAsync()
+        {
+            return await _context.Notification
+                .Where(n =>
+                    n.TreatmentEvents.OrderByDescending(t => t.EventDate).FirstOrDefault().TreatmentEventTypeIsOutcome
```

Comment:
This is currently evaluated locally and not translated to SQL, this should be okay as it's an overnight job however it might be able to be done with raw SQL if that is preferable. FWIW it is run at a different time to the data quality alerts job which also evaluates a bunch of notifications in memory

Reply:
As discussed in person, I think this should potentially be scoped down to notified cases only.
Additionally when doing this we can filter (in SQL) for notification status, and as suggested by you potentially whether there are any treatment outcomes.